src/app/widgets/sign-up-form/sign-up-form-widget.jsx

```diff
@@ -1,30 +1,66 @@
 import { useState } from 'react';
 import { Form, Button } from 'react-bootstrap';
 import './sign-up-form-widget.scss'
+import logoDibba from "../../../assets/images/logo_login.png"
 
 export const SignUpFormWidget = props => {
 
     const [inputDateType, setInputDateType] = useState("text")
 
+    const [passwordSee, setPasswordSee] = useState(false)
```

Comment:
```suggestion
    const [showPassword, setShowPassword] = useState(false)
    const [showConfirmPassword, setShowConfirmPassword] = useState(false)
    const [showPasswordType, setShowPasswordType] = useState("password")
    const [showConfirmPasswordType, setShowConfirmPasswordType] = useState("password")
```

Curiosidade:
Traducao literal do nome que estava -> senha ver
Traducao literal da nova -> ver senha